command.go

```diff
@@ -632,3 +692,47 @@ func (cmd *ScanCmd) parseReply(rd *bufio.Reader) error {
 
 	return nil
 }
+
+//------------------------------------------------------------------------------
+
+type ClusterSlotInfo struct {
```

Comment:
The redis command is called CLUSTER SLOTS - http://redis.io/commands/cluster-slots